editor/editor_node.h

```diff
@@ -692,8 +692,8 @@ class EditorNode : public Node {
 	void set_docks_visible(bool p_show);
 	bool get_docks_visible() const;
 
-	void set_distraction_free_mode(bool p_enter);
-	bool get_distraction_free_mode() const;
+	void set_distraction_free_mode_enabled(bool p_enter);
+	bool is_distraction_free_mode_enabled() const;
```

Comment:
We don't have super consistent APIs around `bool`s for now, but I think the most common would be `set_distraction_free_mode(p_enabled)` and `is_distraction_free_mode_enabled()`.

Reply:
Sounds good to me. In this case, we should be able to preserve backwards compatibility :slightly_smiling_face: